feature_engine/parameter_checks.py

```diff
@@ -34,3 +34,76 @@ def _define_numerical_dict(dict_: Optional[dict]) -> Optional[dict]:
         raise TypeError("The parameter can only take a dictionary or None")
 
     return dict_
+
+
+def check_input_features(
+        input_features: Union[List, str], all_variables: List
+) -> Union[List, str]:
+    """
+    Checks if input_features is None; otherwise, check whether input_variables
+    is a list of features that exist within transformed dataset.
+
+    Parameters
+    ----------
+    input_features : Union[List, str]
+        The features to check whether they
+
+    all_variables: List
+        All the variables associated with the transformed dataset.
+
+    Raises
+    ------
+    ValueError
+        If input_features is not a list or all the features are not included in the transformed
+        variables.
+
+    Returns
+    -------
+    res: bool
```

Comment:
The result is actually feature_names which is a list. Could we update the docstrings?